contrib/python-bindings/include/triangulation_wrapper.h

```diff
@@ -100,13 +100,156 @@ namespace python
                                              const bool           colorize = false);
 
     /**
+     * Like the previous function. However, here the first argument does not
```

Comment:
i presume these are copy-paste from grid-generator? Maybe here and below use copy-doc, i.e.
```
@copydoc FEEvaluationBase<1,1,Number>::get_gradient()
```

Reply:
Good idea.

Reply:
Actually, I don't know how to get doxygen to show these files. I tried to add the path to `contrib/python-bindings/include` in `options.dox.in` but it didn't work. I am not sure that it's a big deal though since these functions are internal. Users should look at the python documentation using an interactive python shell. The comments in this file are more for developer so it might be OK not to have them in doxygen. What do you think?

Reply:
I agree